CHANGELOG.md

```diff
@@ -1,3 +1,190 @@
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/discount-code-importer@2.0.14 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/discount-code-exporter@3.0.14 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/customer-groups-exporter@2.0.14 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/custom-objects-importer@2.0.14 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/custom-objects-exporter@2.0.14 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/csv-parser-state@2.0.13 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/csv-parser-price@3.0.13 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/category-exporter@2.0.14 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
+  - [#1371](https://github.com/commercetools/nodejs/pull/1371) fix(req-builder): enable setting fuzzyLevel to 0 ([@daern91](https://github.com/daern91))
+
+#### Committers: 1
+
+- Daniel Eriksson ([@daern91](https://github.com/daern91))
+
+## @commercetools/api-request-builder@5.3.2 (2019-07-16)
+
+#### :bug: Bug Fix
+
+- `api-request-builder`
```

Comment:
Almost 200 lines of docs just because of one update to the `api-req-builder` 🐼

Reply:
1. I would always strip committers. We do that manually actually.
2. I am not sure why it creates so many entries even though the package is used among all other packages as a dep.

Reply:
1. _Manually_ - we are clearly not thinking lazily enough 👯‍♂ 
2. Yaa, although, it makes sense as all those packages get bumped when they depend on it. But structurally it would make so much more sense to just list all the packages that are affected by the same change IMO 😕